fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/services/ReplacePropertiesService.java

```diff
@@ -38,8 +37,9 @@
      * @throws MalformedRdfException if malformed rdf exception occurred
      */
     void perform(String txId,
-                String userPrincipal,
-                String fedoraId,
-                String contentType,
-                Model inputModel) throws MalformedRdfException;
+                 String userPrincipal,
+                 String fedoraId,
+                 String contentType,
+                 Model inputModel) throws MalformedRdfException;
+
```

Comment:
Just noticed, this could change could be reverted as the only change was spacing. Sorry I didn't catch this earlier.

Reply:
done.